src/base/static/components/input-form/index.js

```diff
@@ -134,7 +141,10 @@ class InputForm extends Component {
         return memo.set(
           field.name,
           Map({
-            value: "",
+            // If this fields has a "forced_default_value", then its default
+            // value will be set and sent on submission even if the field
+            // remains invisible and is never rendered.
+            value: fieldConfig.get("forced_default_value") || "",
```

Comment:
This is a bit of a hack. The issue here is that if a field is not visible (i.e. because it is restricted to members of a certain group), then it will never render on the form and thus will not initialize fully. So if it has a `default_value`, that value will never be set, and when the form is submitted that default value will not be sent.

To work around this issue here, where we *do* want an invisible field with a default value to be sent to the server, I introduced a `forced_default_value` parameter. I thought this would be safer than making all invisible fields with a regular `default_value` submit that value, since I'm not sure if that would cause problems on other flavors.